specfile/README.md

```diff
@@ -0,0 +1,75 @@
+# Packit requirements for the spec-file library
+
+## Manipulation
+
+- We create a Python object by specifying a path of the spec-file.
```

Comment:
I know that we have to specify the path to spec file sources - should we mention that?

Reply:
Is this right (mainly the default)?
```
- We create a Python object by specifying a path of the spec-file and (optionally) path to spec-file sources
  (the directory with the spec-file).
```